releasenotes/notes/deprecate-pauli-table-fc6dcdb5eeb6e0c4.yaml

```
@@ -0,0 +1,9 @@
+---
+deprecations:
+  - |
+    The :class:`~qiskit.quantum_info.PauliTable` and :class:`~qiskit.quantum_info.StabilizerTable`
+    are deprecated and will be removed in a future release.
+    Instead, the :class:`~qiskit.quantum_info.PauliList` should be used.
+    With this change, :meth:`~qiskit.quantum_info.Clifford.table` has been deprecated
+    so that you should operate directly from :meth:`~qiskit.quantum_info.Clifford.tableau`
+    without it.
```

It might be nice to explain why these attributes are deprecated even though they didn't emit a `PendingDeprecationWarning` first. Also we could add the other deprecated attributes 🙂 
```suggestion
    With this change, :attr:`.Clifford.table`, :attr:`.Clifford.stabilizer` 
    and :attr:`.Clifford.destablizer` have been deprecated, as they operate on the
    now deprecated stabilizer table. Instead, use :attr:`.Clifford.tableau`,
    :attr:`.Clifford.stab` and :attr:`.Clifford.destab` as replacements.
```

---

`PendingDeprecationWarning` isn't part of the Qiskit deprecation policy - we don't usually emit it (and almost nobody would see it if we did).

---

Oh I thought it was -- but indeed it only says we need the new path in place and the pending deprecation is optional: https://qiskit.org/documentation/deprecation_policy.html#removing-a-feature Thanks for the correction! 😄

---

Thank you! @jakelishman @Cryoris